src/utils.py

```diff
@@ -63,4 +63,49 @@ def clean_up():
     folder_path = "/code/audio"
     for file in os.listdir(folder_path):
         if file.endswith(".mp3"):
-            os.remove(os.path.join(folder_path, file))
\ No newline at end of file
+            os.remove(os.path.join(folder_path, file))
+
+def get_prompt_templates():
```

Comment:
Perhaps if this is a function just returning constants, we can have place them as constants in the file instead of wrapping it around a function? The function wrapping seems a bit redundant here.

Reply:
Alternatively, would it be considered messy if it were to be left in the llm.py file?